cmd/dalc/main.go

```diff
@@ -7,10 +7,10 @@ import (
 
 	"github.com/celestiaorg/celestia-app/app"
 	"github.com/celestiaorg/dalc/config"
+	"github.com/celestiaorg/dalc/cosmoscmd"
 	"github.com/celestiaorg/dalc/server"
 	"github.com/cosmos/cosmos-sdk/client/keys"
 	"github.com/spf13/cobra"
-	"github.com/tendermint/spm/cosmoscmd"
```

Comment:
Not sure if you chatted with @marbar3778  about this but wouldn't it make sense to merge https://github.com/celestiaorg/celestia-app/pull/177 first and the approach taken there instead of copy&pasting the cosmoscmd code?

Reply:
I see! Thanks for the clarification. I was not sure. It's OK to push this to a follow-up PR if others are blocked on this PR to land btw.